calcom/cal.com
# feat: :sparkles: support for adding script url added for umami

## What does this PR do?

- Adds support for adding the script URL while setting up Umami App.
- The current version of the app only works with hosted solution but with this, users can use their self-hosted version of Umami as well to set up Analytics in cal.com (ref: https://github.com/calcom/cal.com/pull/14783#issuecomment-2104420069)



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- Add the Umami App to your booking.
- Set Script URL along with Site ID.
- Verify the events coming in on your Umami app when the event page is opened.

![SCR-20240821-l8q](https://github.com/user-attachments/assets/b56b4e00-3702-484b-b43d-47baefc49960)

![SCR-20240821-l9e](https://github.com/user-attachments/assets/8b8500e0-0e0a-4e27-ac60-f8f690d3d585)

![SCR-20240821-l9w](https://github.com/user-attachments/assets/dbc28f2a-4a9d-460b-8342-1d6d5fffd3ed)

## Checklist



- I haven't checked if my changes generate no new warnings
